.circleci/config.yml

```diff
@@ -102,6 +102,16 @@ jobs:
             if [[ "$CIRCLE_BRANCH" =~ release-.* ]] || [[ "$CIRCLE_BRANCH" =~ .*-aws-test ]] || [[  "$CIRCLE_BRANCH" == 'master' ]] || [[ ! -z "$RUN_INTEGRATION" ]]; then
               sudo chown -R circleci:circleci /usr/local/bin
 
+              # Build test processing task, CHANGE WHEN PUBLISHED TO NPM
```

Comment:
So we'll be able to remove these as soon as it get's merged?

Reply:
When the 1.6 release is published.

Reply:
I'm planning on doing the release shortly after merging